File: Data Structure/1.Array/Max Consecutive Ones.java
//https://leetcode.com/problems/max-consecutive-ones/
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max = 0;
        int cnt= 0;
        
        for(int i=0;i<nums.length;i++){
            if(nums[i] == 1){
                cnt++;
            }
            else{
                max = Math.max(cnt,max);
                cnt=0;
            }
        }
        
        if(cnt!=0){
            max = Math.max(cnt,max);
        }
        return max;
    }
}

File: Data Structure/1.Array/MaxConsecutiveOnes.py
# https://workat.tech/problem-solving/practice/max-consecutive-ones
class Solution:
	def getMaxConsecutiveOnes(self, arr: List[int]) -> int:
		# add your logic here
		n = len(arr)
		cnt = 0
		maximum = 0
		for i in range(n):
			if(arr[i]==1):
				cnt+=1
			else:
				maximum = max(cnt,maximum)
				cnt =0
		maximum = max(cnt,maximum)
		return maximum